### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by bacboc**

### CODE REVIEW

File: docs/source/public_server.rst [hunk 1]
```diff
@@ -36,9 +36,9 @@ This document describes how you can
 
 
 
-.. _ZeroMQ: http://zeromq.org
+.. _ZeroMQ: https://zeromq.org
 
-.. _Tornado: http://www.tornadoweb.org
+.. _Tornado: https://www.tornadoweb.org
```
Comment (minrk) [inline docs/source/public_server.rst]: These two links cannot be accessed via https

Comment (bacboc) [reply docs/source/public_server.rst]: Oh. I'm sorry, I'll revise it and send you a PR.

Comment (takluyver) [reply docs/source/public_server.rst]: It looks like you've worked this out already, but there's no need to make a new PR - when you push on the same branch, this PR updates automatically.

**Comment by bacboc** [comment]
It is now a step in the beginning and is immature. So now I know. I will do that in the future. Thanks for letting me know.

**Comment by takluyver** [comment]
Thanks!

**Pull request merged**